TROUBLESHOOTING.md

```diff
@@ -119,6 +119,18 @@ If your model shows bad performance, keep in mind the following caveats:
    * When dataset is small, and there are too few steps, this may result in a no-op epoch. Therefore, it is better to use
    small batch sizes in those cases.
 
+## XLA Tensor Quirks
+
+1. **XLA tensor internals are opaque.** XLA tensors always appear to be contiguous and without storage. Networks should not try to check the strides of XLA tensors.
+
+1. **XLA tensors should be moved to the CPU before saving them.** XLA tensors lose their view relationships, if any, when saved. When loaded they are
+placed back on the device they were saved from, but if this device is
+unavailable the load will fail. Moving XLA tensors to the CPU before saving them
```

Comment:
Moving tensors across devices actually loses view information. I think we need to rephrase this sentence.

Reply:
You're right this is very confusing. Updating.